sphinx/texinputs/sphinxhowto.cls

```diff
@@ -93,7 +93,7 @@
 \newenvironment{sphinxthebibliography}[1]{%
   \phantomsection
```

Comment:
@tk0miya remark: actually this `\phantomsection` is now unneeded see http://tex.stackexchange.com/questions/44088/when-do-i-need-to-invoke-phantomsection#comment166081_44091, because indeed the thebibliography environment will do `\section*`